src/app-nordigen/services/nordigen-service.js

```diff
@@ -202,6 +202,13 @@ export const nordigenService = {
     const sortedPendingTransactions = bank.sortTransactions(
       transactions.transactions?.pending,
     );
+    const allTransactions = sortedBookedTransactions.map((t) => {
+      return { ...t, booked: true };
+    });
+    sortedPendingTransactions.forEach((t) =>
+      allTransactions.push({ ...t, booked: false }),
+    );
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
```suggestion
    const allTransactions = [
      ...transactions.transactions?.booked.map((t) => {
        return { ...t, booked: true };
      }),
      ...transactions.transactions?.pending.map((t) => {
        return { ...t, booked: false };
      }),
    ];
```

Reply:
I did it like this mostly because my way prevents unnecessary double iteration over the array but I imagine it's not a big deal :smile:

Reply:
Both solutions have double iteration btw.

But yea.. this is mostly just a nitpick :)

Reply:
I'll update this but could you tell me if there's any reason to prefer `transactions.transactions?.booked` over `sortedBookedTransactions` here?

Reply:
oh, you meant the spread operation. Yea, that's sub-optimal in this approach.

Reply:
> I'll update this but could you tell me if there's any reason to prefer transactions.transactions?.booked over sortedBookedTransactions here?

I imagine we'll want to remove `sortedBookedTransactions` a few releases later in favor of the new `all`. So it will make the future migration a bit easier. But again: no biggie. It's just a small nitpick. :)

Reply:
> oh, you meant the spread operation. Yea, that's sub-optimal in this approach.

Yep. I opted for a more performant way first and left it up to you to either tell me to use the cleaner way or the optimal way. But to be honest, optimizing this is a bit nitpicky on my side since I don't imagine it could possibly have any measurable difference with the number of transactions we could get at most.